simapp/app/abci.go

```diff
@@ -0,0 +1,84 @@
+package simapp
+
+import (
+	"bytes"
+	"crypto/rand"
+	"encoding/json"
+	"fmt"
+
+	abci "github.com/cometbft/cometbft/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type (
+	// VoteExtensionHandler defines a dummy vote extension handler for SimApp.
+	//
+	// NOTE: This implementation is solely used for testing purposes. DO NOT use
+	// in a production application!
+	VoteExtensionHandler struct{}
+
+	// VoteExtension defines the structure used to create a dummy vote extension.
+	VoteExtension struct {
```

Comment:
can we extend votes with commonware? 

if not i think we shoud kill it and use a different approach as it could cause issues with signature aggregation